vendor/nexelec/feel-profile.yaml

```diff
@@ -1,13 +1,13 @@
 # LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
-macVersion: 1.0.2
+macVersion: 1.0.4
 # LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
 #   1.0:   TS001-1.0
 #   1.0.1: TS001-1.0.1
 #   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
 #   1.0.3: RP001-1.0.3-RevA
 #   1.0.4: RP002-1.0.0 or RP002-1.0.1
 #   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
-regionalParametersVersion: RP001-1.0.2-RevB
```

Comment:
Same goes for the parameters.

Reply:
Hi 

I've taken your comments into consideration
I went back to the initial settings for the products 
Feel
Rise
Move
Sign